lib/devise/passkeys/reauthentication_strategy.rb

```diff
@@ -5,12 +5,12 @@
 
 module Devise
   module Strategies
-    class PasskeyReauthentication < PasskeyAuthenticatable
+    class Reauthentication < PasskeyAuthenticatable
       def authentication_challenge_key
         "#{mapping.singular}_current_reauthentication_challenge"
       end
     end
   end
 end
 
-Warden::Strategies.add(:passkey_reauthentication, Devise::Strategies::PasskeyReauthentication)
+Warden::Strategies.add(:Reauthentication, Devise::Strategies::Reauthentication)
```

Comment:
This needs to be reverted back to `:passkey_reauthentication`

```suggestion
Warden::Strategies.add(:passkey_reauthentication, Devise::Strategies::Reauthentication)
```